src/rust/engine/sandboxed_cargo.sh

```diff
@@ -0,0 +1,9 @@
+# THis simply exists because `shell_command` at the time of writing doesn't know how to merge `PATH`
+# from the rule's env and the `extra_env_vars`.
+
+PYTHON_PATH=${CHROOT}/3rdparty/tools/python3/python/bin
```

Comment:
Shell style nit: Surround all of the values in double quotes in case any of the variable substitutions have a space in them.